Doppler.Integrations/appsettings.Integration.json

```diff
@@ -1,6 +1,5 @@
 ﻿{
   "DopplerAPI": {
-    //"Base_URL": "http://newapiint.fromdoppler.net/"
-    "Base_URL": "https://restapi.fromdoppler.com/"
+    "Base_URL": "http://newapiint.fromdoppler.net/"
```

Comment:
I think this should be reverted, right?

Reply:
Is the config for Integration enviroment